lib/utils.js

```diff
@@ -35,7 +36,7 @@ function mkdirpSync (dirpath) {
   }
 }
 
-function runCommand (command, args, ignoreStdError) {
+function runExecutable (command, args, ignoreStdError) {
   return new RSVP.Promise(function (resolve, reject) {
     var child = spawn(command, args, {
```

Comment:
@EWhite613 I haven't touch calls to `git` in this pr. So I think the issue still persist for `teamcity-bitbucket-visual-acceptance` and `travis-visual-acceptance` commands.

Anyway the goal of this pr is to enable running of `ember` commands via `lookup-command`. I believe we should fix `git` in a separate pr.